test/test_dos.py

```diff
@@ -5030,11 +5030,9 @@ def setUpClass(cls):
 
     def setUpDosConfig(self):
         # Use the (almost) standard shipped config
-        with open(join("src/bindist", self.confsys), "r") as f:
-            contents = f.read()
-            contents = re.sub(r"SWITCHES=#0", r"SWITCHES=/F", contents)
-            self.mkfile(self.confsys, contents, newline="\r\n")
-
+        contents = (self.topdir / "src" / "bindist" / self.confsys).read_text()
+        contents = re.sub(r"SWITCHES=#0", r"SWITCHES=/F", contents)
```

Comment:
This is for non-fdpp, right?

Reply:
Hmm, for non-fdpp you'd be using
the specific config.sys, so I suppose
this IS for fdpp? Why?

Reply:
This is in the FDPP TestCase setup and serves only to set the `/F` switch for each test, as I think you said it made the startup faster?

Reply:
/F makes it faster only on
non-fdpp, because on fdpp
it is a default. On fdpp you
can do "dosemu -Y" to enter
interactive mode, so this
start-up delay is unneeded
and is disabled.

Reply:
Okay, I'll replace this change with one that removes the fdppconf.sys modification.